docs/tutorials/TracingCode.md

```diff
@@ -180,7 +180,7 @@ Recall from the User Guide that the `edit` command has the format: `edit INDEX [
     * {@code JsonSerializableAddressBook}.
     */
    public JsonSerializableAddressBook(ReadOnlyAddressBook source) {
```

Comment:
Maybe change this to `JsonSerializableStudentList` and correspondingly `ReadOnlyStudentList`?